index.bs

```diff
@@ -2916,6 +2916,246 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's `signal()` Method ### {#sctn-signal}
```

Comment:
I don't necessary have a better suggestion other than the previously attempted `report,` but `signal` as a name is already used in CredentialRequestOptions/CredentialCreationOptions in Credential Management which makes this a little odd. I almost like "Stale Credentials API," but that might be too specific to the current report types.

Reply:
(renamed methods to signalUnknownCredentialId, etc)